OSDOCS-8890 z-stream RNs for 4.14.5 (#68769)
openshift/openshift-docs: release_notes/ocp-4-14-release-notes.adoc

====

[id="ocp-4-14-5"]
=== RHSA-2023:7599 - {product-title} 4.14.5 bug fix and security update

Issued: 2023-12-05

{product-title} release 4.14.5, which includes security updates, is now available. The list of bug fixes that are included in the update is documented in the link:https://access.redhat.com/errata/RHSA-2023:7599[RHSA-2023:7599] advisory. The RPM packages that are included in the update are provided by the link:https://access.redhat.com/errata/RHBA-2023:7603[RHBA-2023:7603] advisory.

Space precluded documenting all of the container images for this release in the advisory.

You can view the container images in this release by running the following command:

[source,terminal]
----
$ oc adm release info 4.14.5 --pullspecs
----

[id="ocp-4-14-5-bug-fixes"]
==== Bug fixes

* Previously, when the build capability is not enabled, ConfigObserver controller would fail when trying to sync with build informer. With this release, ConfigObserver starts successfully when build capability is not enabled.
(link:https://issues.redhat.com/browse/OCPBUGS-23490[*OCPBUGS-23490*])

> * Previously, updating the VMware vCenter server in root secret vSphere-creds in the `kube-system` namespace was not supported by the Cloud Credential Operator (CCO). This meant that the component secrets were not synchronized correctly. With this release, the CCO will reset the secret data when synchronized. CCO now supports updating the VMware vCenter server in root secret vSphere-creds in the `kube-system` namespace. (link:https://issues.redhat.com/browse/OCPBUGS-23426[*OCPBUGS-23426*])

Reviewer: Keeping terms more in-line with [CCO docs](https://68769--docspreview.netlify.app/openshift-enterprise/latest/authentication/managing_cloud_provider_credentials/about-cloud-credential-operator)

```suggestion
* Previously, the Cloud Credential Operator (CCO) did not support updating the VMware vSphere root secret (`vsphere-creds`) in the `kube-system` namespace. This prevented the component secrets from synchronizing correctly. With this release, the CCO supports updating the vSphere root secret and resets the secret data when synchronized. (link:https://issues.redhat.com/browse/OCPBUGS-23426[*OCPBUGS-23426*])
```

[id="ocp-4-14-5-updating"]
==== Updating

> To update an existing {product-title} 4.14 cluster to this latest release, see xref:../updating/updating_a_cluster/updating-cluster-cli.adoc#updating-cluster-cli[Updating a cluster using the CLI].

Reviewer 2: ```suggestion
To update an existing {product-title} 4.14 cluster to this latest release, see xref:../updating/updating_a_cluster/updating-cluster-cli.adoc#updating-cluster-cli[Updating a cluster by using the CLI].
```
See ISG: https://www.ibm.com/docs/en/ibm-style?topic=word-usage#u
Because this is an existing module title, you can see if this change can quickly be done now or not.

Author: Why would this be a problem? "By using" is acceptable per the style guide in the 'using' entry, yes?

Reviewer 3: It's currently just "using."
